docs/deploy.md

```diff
@@ -26,26 +26,44 @@ You'll need them in the steps to come. The portal's name should be concise and d
 
 1. Use Heroku's automatically generated URL.
 
-It will look like this: `[name of your app].herokuapp.com`. While this is convenient and easy, it needs to be unique among other Heroku apps and it may not be the easiest for your volunteers to remember.
+  It will look like this: `[name of your app].herokuapp.com`. While this is convenient and easy, it needs to be unique among other Heroku apps and it may not be the easiest for your volunteers to remember.
 
 2. Use a custom domain name.
 
-You can connect any custom domain name you own to your Heroku deployment. We'll go over how to link the two later in the setup, but make sure you know what it is!
+  You can connect any custom domain name you own to your Heroku deployment. We'll go over how to link the two later in the setup, but make sure you know what it is!
 
-### Set up Google OAuth
+### Set up User Authorization
+
+Volunteer Portal currently supports two forms of user authentication:  Google Oauth and SAML.  Choose whichever one makes most sense for you, and follow the instructions below to get started.  If you have requests for more authentication methods, feel free to let us know by joining our Slack channel ([the link is in our main README](https://github.com/zendesk/volunteer_portal/blob/ahart/okta-auth/README.md#volunteer-portal)), or follow [our developer documentation](https://github.com/zendesk/volunteer_portal/blob/ahart/okta-auth/docs/development.md) to add it yourself!
```

Comment:
probably don't want to link to this branch, just use relative URLs for these links.